tests/test_parallel_upload_download.cpp

```diff
@@ -115,6 +115,18 @@ void test_parallel_upload_core(int fileNumber)
   }
   assert_int_equal(status, SF_STATUS_EOF);
 
+  std::string getCommand = "get @%test_parallel_upload_download file://" + dataDir + "test_parallel_download";
```

Comment:
where is dataDir? can multiple tests run in a single machine?

Reply:
dataDir is under <workspace>/libsnowflakeclient/tests/data/

It is fine as long as each test run has its own local git repo.

Reply:
ok, then add the file name to .gitignore such that somebody accidentally add to the repo.